src/Tests/TestCommon/Requirements.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.Azure.SignalRBench.Tests
+{
+    public static class Requirements
```

Comment:
How about use `Atttibute` like [`SkipIfConnectionStringNotPresentAttribute`](https://github.com/Azure/azure-signalr/blob/d9bea3cce0df358f6b35c9f5d88969c1dde6f25e/test/Microsoft.Azure.SignalR.Tests.Common/SkipIfConnectionStringNotPresentAttribute.cs). It looks clearer

Reply:
I cannot find that package in nuget.org